app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,10 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "main", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{ENV['MAPS_API_KEY']}&libraries=places&callback=dispatchMapsEvent",
```

Comment:
Also remember to leave the example file, where your environment variable (MAPS_API_KEY) appears, so that any user who downloads your project knows how to configure the project.

Reply:
Fixed!